### PR TITLE
Add send results by email task

### PR DESCRIPTION
**Pull request by AlexFalessi**
Add a task that can send results array content and a log file by email.

### CODE REVIEW

File: Email/resources/catalog/Send_Results_By_Email.xml [hunk 1]
```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.12" xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"  name="Send_Results_By_Email" projectName="Notifications" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="1"  >
+  <description>
+    <![CDATA[ A task that will send direct parent tasks results by email. ]]>
+  </description>
+  <genericInformation>
+    <info name="bucketName" value="basic-examples"/>
```
Comment (mboussaa) [inline Email/resources/catalog/Send_Results_By_Email.xml:9]: shouldn't be in `notification tools`?

File: Email/resources/catalog/Send_Results_By_Email.xml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    <info name="workflow.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/email_notification_red_one.png"/>
```
Comment (mboussaa) [inline Email/resources/catalog/Send_Results_By_Email.xml:10]: no doc link ?

File: Email/resources/catalog/Send_Results_By_Email.xml [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        <variable name="to" value="user@domain.org" inherited="true" model="PA:NOT_EMPTY_STRING"/>
+      </variables>
+      <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/email_notification_red_one.png"/>
+      </genericInformation>
+      <scriptExecutable>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+// Sending an email requires to configure an SMTP connection
```
Comment (mboussaa) [inline Email/resources/catalog/Send_Results_By_Email.xml:36]: please use /*  */ for all ur multiline comments. Don't use //

File: Email/resources/catalog/Send_Results_By_Email.xml (hunk 1 above)
Comment (AlexFalessi) [reply Email/resources/catalog/Send_Results_By_Email.xml:9]: You are right, I will update. Thanks

**Pull request merged**